src/js/api/errors.js

```diff
@@ -24,6 +24,11 @@ export const SETUP_ERROR_LICENSE_INVALID = 100012;
  */
 export const SETUP_ERROR_LICENSE_EXPIRED = 100013;
 
+/**
+ * @enum {ErrorCode} Setup failed because a core module failed to load.
+ */
+export const SETUP_ERROR_LOADING_CORE_JS = 101000;
```

Comment:
@robwalch @johnBartos earlier we agreed that we would export 6 digit codes, but if a base code or a category code were to change we wouldn't want to have to update all the codes that are built off of it. How do you guys feel about
```
const SETUP_ERROR_BASE_CODE = 100000;
export const SETUP_ERROR_LOADING_CORE_JS = SETUP_ERROR_BASE_CODE + 1000;
export const SETUP_ERROR_LICENSE_EXPIRED = SETUP_ERROR_BASE_CODE + 13;
```

Reply:
I think this is over doing it, and prevents someone from searching our code base for some of the specific error codes.

We should not update base codes. New errors can have new codes. Old ones should not change.